raiden/transfer/channel.py

```diff
@@ -1184,10 +1184,22 @@ def get_max_withdraw_amount(
     sender: NettingChannelEndState, receiver: NettingChannelEndState
 ) -> WithdrawAmount:
     """Calculates the maximum "total_withdraw_amount" for a channel.
-    This will leave the channel without funds, when
+    This will leave the channel without funds, when this is withdrawn from the contract,
+    or pending as offchain-withdraw.
 
     """
-    return WithdrawAmount(sender.total_withdraw + get_balance(sender, receiver))
```

Comment:
This doesn't seem right. Why would we not need to include the previous off-chain withdraws in the calculation (which is what the `get_balance()` function does)?

Reply:
Ok, actually this is correct since coop settle requires no pending withdraws.
However I still think it would be nicer to use `get_balance` instead of ad-hoc reimplementing half of it here.

Reply:
Done.